docs/docs/recipes/integrate-logto/components/AppNote.tsx

```diff
@@ -8,8 +8,10 @@ type Props = {
 const AppNote = ({ type }: Props) => {
   return (
     <Admonition type="note">
-      This tutorial assumes you have created a {type} Application in Admin Console. If you are not
-      ready, read this one before continuing.
+      This tutorial assumes you have created an Application of type &quot;{type}&quot; in Admin
+      Console. If you are not ready,{' '}
+      <a href="../../tutorials/get-started/create-and-integrate-the-first-app">read this</a> one
```

Comment:
The link should be mounted to `read this one` or `this one`? Or maybe you can simply remove `one`.

Reply:
yea I refactored in another PR